### ISSUE TITLE
Remove `text-transform: uppercase` from buttons

### ISSUE THREAD

**Issue by caosborne89** [az-digital/arizona-bootstrap#1245]
## Motivation
All the button text styles are uppercase. It would be useful to add alternatives for each button style that removed the uppercase text-transformation.

## Proposed Resolution
~~Add alternatives for each button style that does not include the uppercase text-transformation.~~

Updated approach: Update the default appearance of buttons to not include `text-transform: uppercase`, and include the change as part of the next minor release of AZ Bootstrap. We should also backport to 2.x.

**Comment by joshuasosa** [comment]
Sentence case is what Bootstrap uses by default:
https://getbootstrap.com/docs/4.6/components/buttons/
https://getbootstrap.com/docs/5.2/components/buttons/

Even if options are added, and a new class created, perhaps there should be consideration about changing the default off of all caps.

More reading:
- https://www.w3.org/TR/low-vision-needs/#capitalization
- https://thestory.is/en/journal/all-caps-in-ux-design/

**Comment by akslay** [comment]
Bringing over some relevant notes from the conversations on this thread: https://uarizona.slack.com/archives/C0SAW6H4J/p1719869500798899

- We'd like to change the default behavior but felt that was a change that probably necessitates a minor/major version bump of Arizona Bootstrap (which is something we haven't really done so far).
- Should coincide with the release of AZ Quickstart 2.12.
- Still need to decide how best to merge this change into AZ Bootstrap, but PR should be made against `main` branch (where all `5.x` changes are too).
- How to include that change into AZBS 2.x release (based on Bootstrap 4.x) will be a more complicated thing we'll need to figure out.

**Comment by mmunro-ltrr** [comment]
Backporting to the 2.x branch sounds like a good idea, too.
